alluxio/alluxio_file_system.py

```diff
@@ -127,10 +127,18 @@ def __init__(
 
         # parse worker info to form hash ring
         worker_addresses = None
-        if etcd_host:
-            worker_addresses = EtcdClient(
-                host=etcd_host, options=options
-            ).get_worker_addresses()
+        if etcd_hosts:
+            worker_addresses = []
+            for host in etcd_hosts.split(","):
+                try:
+                    worker_addresses = EtcdClient(
```

Comment:
Users are likely to input ETCD client as the same order e.g. host1,host2,host3
in your approach, all requests will always go to host1 first, then host2, then host3
can you improve to give all the hosts same chances to receive the client request?